sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -931,6 +931,113 @@ private void SetNameFieldsIfNull()
         }
         #endregion Delete
 
+        #region Exists
+        /// <summary>
+        /// The <see cref="ExistsAsync"/> operation can be called on a
+        /// <see cref="BlobContainerClient"/> to see if the associated container
+        /// exists on the storage account in the storage service.
+        /// </summary>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
+        /// <returns>
+        /// A <see cref="Response"/> if successful.
```

Comment:
Should this say Response{bool} indicating whether the container exists or not?
Also similar comment for BlobBaseClient.

Reply:
fixed.